identity/backend/src/main/java/io/camunda/identity/usermanagement/model/Group.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.usermanagement.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "groups")
+public class Group {
+  @Id private Long id;
```

Comment:
❓ Does this also need `@GeneratedValue`? I'm not sure if `@Id` handles generation

Reply:
Well, right now we don't create the group here, it all done in UserDetailsManagers and group entity is defined for query (in better word to not forced to use native queries).